src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -285,6 +285,29 @@ We added 2 npm scripts to help run these tools:
 - ``npm run lint``, will check frontend files against all linters
 - ``npm run fix``, will try to fix all the detected linting errors
 
+Ceph Dashboard and Bootstrap
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Currently we are using Bootstrap 4.5 on the Ceph Dashboard. This means that most of our SCSS and HTML code can make use
```

Comment:
In order to make this section more sustainable, I wouldn't mention the exact version of Bootstrap.
@LenzGr what's your take on this?

Reply:
@SchoolGuy @LenzGr It seems the last paragraph (```To change the variable defaults...```) of this section has to be updated:
https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/HACKING.rst#frontend-branding

Reply:
Indeed, it's easy to forget to change version numbers in such "hidden" README files.
Would it make sense to point to `package.json` instead for the exact version number?

"Currently, we are using the Bootstrap toolkit on the Ceph Dashboard (see the file package.json for the current version used). This means..."

Reply:
> @SchoolGuy @LenzGr It seems the last paragraph (`To change the variable defaults...`) of this section has to be updated:
> https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/HACKING.rst#frontend-branding

I created this tracker issue for this: https://tracker.ceph.com/issues/46869

Reply:
I adjusted the paragraph regarding the Bootstrap version. Thus please unresolve this discussion if you think there needs to be done more.